src/js/form-builder.js

```diff
@@ -1713,6 +1718,11 @@ function FormBuilder(opts, element, $) {
           selectedOption.checked = prevOptions[i].checked
         })
       }
+    } else if (field.type === 'textarea') {
+      const fieldVal = document.getElementById('value-' + field.id)
+      if (fieldVal) {
+        fieldVal.innerHTML = e.target.value
+      } 
```

Comment:
```suggestion
```

Why have you chosen to convert between raw string value and HTML? We can pass the value directly between the value get/set for the elements `fieldVal.value = e.target.value`

Reply:
In the previous set I was using the contenteditable which meant that the helper.js function getAttrVals function was using the innerHTML.  Since you amendment removed that I agree no point to this piece of code anymore.